Source/Test/CompanyCommunicator.Test/Controllers/AuthenticationMetadataControllerTest.cs

```diff
@@ -47,7 +47,7 @@ public void CreateInstance_AllParameters_ShouldBeSuccess()
         {
             // Arrange
             this.options.Setup(x => x.Value).Returns(new AuthenticationOptions() { AzureAdTenantId = this.tenantId, AzureAdClientId = this.clientId });
-            Action action = () => new AuthenticationMetadataController(this.options.Object);
+            Action action = () => new AuthenticationMetadataController(this.options.Object, null /*TODO - fix this*/);
```

Comment:
what's the resolution for these TODOs

Reply:
I fixed all the build issues in the test projects. These tests need to be updated. Have asked the template team to help with that effort.